### ISSUE TITLE
Blurry on high dpi monitor

### ISSUE THREAD

**Issue by colemickens** [kennylevinsen/wldash#20]
Zoom in on the screenshot and you'll see.
![output](https://user-images.githubusercontent.com/327028/64997714-630cfb80-d896-11e9-8808-73d23e08b362.png)

**Comment by kennylevinsen** [comment]
Yeah, DPI should probably be handled automatically in the future. However, for the moment, you can set it in the config: `scale: 2` (assuming your dpi scale is 2).

The layout becomes small, but that's just a matter of configuring the sizes.
